e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -41,6 +41,14 @@ Scenario('HMCTS super user updates FamilyMan reference number', async ({I, caseV
   I.seeFamilyManNumber('newMockCaseID');
 });
 
+Scenario('HMCTS super user updates language requirement', async ({I, caseViewPage, enterLanguageRequirementsEventPage}) => {
```

Comment:
going by the event defs this shouldn't be in the super user test. the event defs only give permissions to `caseworker-publiclaw-courtadmin` which the superuser that we have made just so happens to have. Might be better putting it in `hmctsAdministersCaseAfterSubmission_test.js`